src/ucs/datastruct/ptr_array.c

```diff
@@ -196,29 +198,41 @@ unsigned ucs_ptr_array_insert(ucs_ptr_array_t *ptr_array, void *value,
 
     elem = &ptr_array->start[index];
 
-    /* Remove from free list and Populate */
+    /* Remove from free list and populate */
     ptr_array->freelist = ucs_ptr_array_freelist_get_next_params(*elem,
-        placeholder_p);
+                                                             &size_free_ahead);
```

Comment:
still unaligned

Reply:
If I put more spaces there, the next line will be longer than 80 characters - What's the preference in this case?

Reply:
OK, so I fixed differently - I added space after the '=', as follows:
    ptr_array->freelist =
        ucs_ptr_array_freelist_get_next_params(*elem, &size_free_ahead);